Header header
float32[] force_x
float32[] force_y
float32[] force_z
float32[] obst_x
float32[] obst_y
float32[] obst_z
float32 pose_x
float32 pose_y
float32 pose_z 

 



